openpos-client-libs/projects/openpos-client-core-lib/src/lib/core/platform-plugins/enterprise-config/enterprise-config.interface.ts

```diff
@@ -0,0 +1,32 @@
+import {Observable} from "rxjs";
+
+export interface IEnterpriseConfig {
+
+    /**
+     * Unique name to identify this configuration
+     */
+    name(): string;
```

Comment:
As a method I would expect this to be a verb, like `getName(): string`. In this case I would expect it to be a readonly field

```suggestion
    readonly name: string;
```

Reply:
Sure, I'll adjust this.

Reply:
I'm not going to make this change because `platform-plugin.interface.ts` already has a `name()` method and I purposefully want to have the flexibility for the implementations of the interface to both a IPlatformPlugin and a IEnterpriseConfig.  typescript will not allow me to have both a `name` attribute and a `name()` method.